etcdctl/ctlv3/ctl.go

```diff
@@ -42,10 +45,14 @@ var (
 		Use:        cliName,
 		Short:      cliDescription,
 		SuggestFor: []string{"etcdctl"},
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			HandleConfigFile(cmd)
+		},
 	}
 )
 
 func init() {
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.ConfigFile, "config", "c", "", "yaml config file with global flags")
```

Comment:
server uses config-file naming we should probably align?

Reply:
consider aligning with server

```
Path to the client configuration file. Note that if a configuration file is provided, other command line flags and environment variables will be ignored.
```

Reply:
Changed to `config-file`. I thought that leaving the existing precedence model was better, but I'm open to suggestions.